<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckboxTrackListingDialog</class>
 <widget class="QDialog" name="CheckboxTrackListingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Tracks to Remove</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>631</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="navigationButtonLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="leftButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>←</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rightButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>→</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="okCancelbuttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>861</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="instructionsLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>821</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>The following tracks are SIMILAR to each other by artist and title. Select the track(s) you want to remove:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pageNumberLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>490</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Page</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelbuttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CheckboxTrackListingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelbuttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CheckboxTrackListingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
